azext_iot/monitor/models/arguments.py

```diff
@@ -0,0 +1,69 @@
+# coding=utf-8
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+from azure.cli.core.commands import AzCliCommand
+from azext_iot.common.utility import init_monitoring
+from azext_iot.monitor.models.enum import Severity
+
+
+class TelemetryArguments:
+    def __init__(
+        self,
+        cmd: AzCliCommand,
+        timeout: int,
+        properties: list,
+        enqueued_time: int,
+        repair: bool,
+        yes: bool,
+    ):
+        (enqueued_time, unique_properties, timeout_ms, output) = init_monitoring(
+            cmd, timeout, properties, enqueued_time, repair, yes
+        )
+        self.output = output
+        self.timeout = timeout_ms
+        self.properties = unique_properties
+        self.enqueued_time = enqueued_time
+
+
+class CommonParserArguments:
+    def __init__(
+        self, properties=[], interface_name="", content_type="",
```

Comment:
Let's not use any mutable objects as default values for function parameters in order to avoid issues later on. properties can be `None` here.